www/importentity.php

```diff
@@ -26,7 +26,7 @@
 $as = new SimpleSAML_Auth_Simple($authsource);
```

Comment:
So we instantiate $as with an `$authsource` (where does that come from?) and right below now pass `$authenticationSource`. Is that really correct?